Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -190,8 +190,9 @@ void CNANDContentLoader::InitializeContentEntries(const std::vector<u8>& tmd, co
 		content.m_Index     = Common::swap16(&tmd[entry_offset + 0x01E8]);
 		content.m_Type      = Common::swap16(&tmd[entry_offset + 0x01EA]);
 		content.m_Size      = static_cast<u32>(Common::swap64(&tmd[entry_offset + 0x01EC]));
-		std::copy(&tmd[entry_offset + 0x01E4], &tmd[entry_offset + 0x01E4 + 36], content.m_Header);
-		std::copy(&tmd[entry_offset + 0x01F4], &tmd[entry_offset + 0x01F4 + 20], content.m_SHA1Hash);
+		// This syntax is used to avoid out of bound
```

Comment:
This is kind of meaningless now. It's a valid form of using copy just as much as using iterators is